src/config.rs

```diff
@@ -878,6 +881,12 @@ impl Config {
             .map(|p| p.as_path())
     }
 
+    pub fn shell(&self) -> Option<&Path> {
+        self.shell
+            .as_ref()
+            .map(|p| p.as_path())
```

Comment:
The other case should probably be updated as well. The closure here is just redundant is all.